understanding/20/section-headings.html

```diff
@@ -11,11 +11,12 @@ <h1>Understanding Section Headings</h1>
    <section id="brief">
       <h2>In brief</h2>
       <dl>
-         <dt>Objective</dt><dd>Users understand how content is organized in sections</dd>
-         <dt>Author task</dt><dd>Where content is organized in sections, provide section headings</dd>
+         <dt>Goal</dt><dd>Users understand how content is organized in sections.</dd>
+         <dt>What to do</dt><dd>Where content is organized in sections, provide section headings.</dd>
+         <dt>Why it's important</dt><dd>People with cognitive disabilities or reduced sight can orient themselves.</dd> 
```

Comment:
correcting to match headings and label
```suggestion
         <dt>Why it's important</dt><dd>People can orient themselves, especially those with cognitive or visual disabilities.</dd> 
```